### PR TITLE
Stardog Integration Metric Edits 

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Add periods to end of metric descriptions and removes []() linkage from the metric descriptions. Links do not render in in-app tiles but they do render on the doc pages.

### Motivation

Chat with Alex Lopez on Slack

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: stardog/metadata.csv [hunk 1]
```diff
@@ -1,507 +1,507 @@
...
-stardog.databases.dbname.equality.numLevels,gauge,,unit,,"The configured number of levels for the equality index. This is set by configuration, and won't change during the lifecycle of the process",0,stardog,dbname equality numLevels,
-stardog.databases.dbname.equality.slowdowns,gauge,,unit,,The total number of slowdowns which have occurred on the equality index since the process started or the database was created. Slowdowns are when the data must be delayed in order to allow compaction to reduce the file count to a give level,0,stardog,dbname equality slowdowns,
+stardog.databases.dbname.equality.memtable.total.size.bytes,gauge,,byte,,"The current size of all memtables(active, inactive, and immutable), in bytes.",0,stardog,total size bytes,
+stardog.databases.dbname.equality.memtable.unpinned.size.bytes,gauge,,byte,,The current size of all unpinned memtables for the equality index. Unpinned memtables are memtables which are not currently pinned in memory for readers,0.,stardog,unpinned size bytes,
```
Comment (alai97) [inline stardog/metadata.csv:209]:
```suggestion
stardog.databases.dbname.equality.memtable.unpinned.size.bytes,gauge,,byte,,The current size of all unpinned memtables for the equality index. Unpinned memtables are memtables which are not currently pinned in memory for readers.,0,stardog,unpinned size bytes,

```

File: stardog/metadata.csv [hunk 2]
```diff
@@ -1,507 +1,507 @@
...
+stardog.databases.dbname.ternary.backgroundErrors,gauge,,unit,,"The number of errors that were detected during background processing of the main index storing encoded data since the process began.",0,stardog,dbname ternary backgroundErrors,
+stardog.databases.dbname.ternary.compaction.keysDropped,gauge,,unit,,The number of keys which were removed as part of the compaction process.,0,stardog,ternary compaction keysDropped,
+stardog.databases.dbname.ternary.compaction.keysProcessed,gauge,,unit,,The number of keys which were processed during compaction.,0,stardog,ternary compaction keysProcessed,
+stardog.databases.dbname.ternary.compaction.memory.total,gauge,,byte,,The total amount of memory currently being used to perform compactions for the main index storing encoded data,0.,stardog,compaction memory total,
```
Comment (alai97) [inline stardog/metadata.csv:291]:
```suggestion
stardog.databases.dbname.ternary.compaction.memory.total,gauge,,byte,,The total amount of memory currently being used to perform compactions for the main index storing encoded data.,0,stardog,compaction memory total,

```

File: stardog/metadata.csv [hunk 3]
```diff
@@ -1,507 +1,507 @@
...
+stardog.databases.dbname.binary.memtable.total.size.bytes,gauge,,byte,,"The current size of all memtables (active, inactive, and immutable), in bytes.",0,stardog,total size bytes,
+stardog.databases.dbname.binary.memtable.unpinned.size.bytes,gauge,,byte,,The current size of all unpinned memtables for the binary count index. Unpinned memtables are memtables which are not currently pinned in memory for readers.,0,stardog,unpinned size bytes,
+stardog.databases.dbname.binary.numFilesCompacting,gauge,,unit,,The current number of files compacting for the binary count index.,0,stardog,dbname binary numFilesCompacting,
+stardog.databases.dbname.binary.numKeys,gauge,,unit,,"The estimated number of keys in the binary count index. For Ternary indices, this is a (rough) estimateo the number of quads in the database, for the dictionary, it's an estimate of how many statements are in the dictionary. Note that this _not_ a transactional estimate: deleted entries are ignored, so this value will likely overcount in that scenario.",0,stardog,dbname binary numKeys,
```
Comment (alai97) [inline stardog/metadata.csv:47]:
```suggestion
stardog.databases.dbname.binary.numKeys,gauge,,unit,,"The estimated number of keys in the binary count index. For Ternary indices, this is a rough estimate of the number of quads in the database, for the dictionary, it's an estimate of how many statements are in the dictionary. Note that this _not_ a transactional estimate: deleted entries are ignored, so this value will likely overcount in that scenario.",0,stardog,dbname binary numKeys,

```

File: stardog/metadata.csv [hunk 4]
```diff
@@ -1,507 +1,507 @@
...
-stardog.databases.dbname.dictionary.value.numLevels,gauge,,unit,,"The configured number of levels for the dictionary decoding table for the database. This is set by configuration, and won't change during the lifecycle of the process",0,stardog,dictionary value numLevels,
-stardog.databases.dbname.dictionary.value.slowdowns,gauge,,unit,,The total number of slowdowns which have occurred on the dictionary decoding table for the database since the process started or the database was created. Slowdowns are when the data must be delayed in order to allow compaction to reduce the file count to a give level,0,stardog,dictionary value slowdowns,
+stardog.databases.dbname.dictionary.value.memtable.total.size.bytes,gauge,,byte,,"The current size of all memtables(active, inactive, and immutable), in bytes.",0,stardog,total size bytes,
+stardog.databases.dbname.dictionary.value.memtable.unpinned.size.bytes,gauge,,byte,,The current size of all unpinned memtables for the dictionary decoding table for the database. Unpinned memtables are memtables which are not currently. pinned in memory for readers,0,stardog,unpinned size bytes,
```
Comment (alai97) [inline stardog/metadata.csv:168]:
```suggestion
stardog.databases.dbname.dictionary.value.memtable.unpinned.size.bytes,gauge,,byte,,The current size of all unpinned memtables for the dictionary decoding table for the database. Unpinned memtables are memtables which are not currently pinned in memory for readers.,0,stardog,unpinned size bytes,

```

File: stardog/metadata.csv [hunk 5]
```diff
@@ -1,507 +1,507 @@
...
-stardog.databases.dbname.dictionary.dict.numLevels,gauge,,unit,,"The configured number of levels for the dictionary encoding table for the database. This is set by configuration, and won't change during the lifecycle of the process",0,stardog,dictionary dict numLevels,
-stardog.databases.dbname.dictionary.dict.slowdowns,gauge,,unit,,The total number of slowdowns which have occurred on the dictionary encoding table for the database since the process started or the database was created. Slowdowns are when the data must be delayed in order to allow compaction to reduce the file count to a give level,0,stardog,dictionary dict slowdowns,
+stardog.databases.dbname.dictionary.dict.memtable.total.size.bytes,gauge,,byte,,"The current size of all memtables(active, inactive, and immutable), in bytes.",0,stardog,total size bytes,
+stardog.databases.dbname.dictionary.dict.memtable.unpinned.size.bytes,gauge,,byte,,The current size of all unpinned memtables for the dictionary encoding table for the database. Unpinned memtables are memtables which are not currently. pinned in memory for readers,0,stardog,unpinned size bytes,
```
Comment (alai97) [inline stardog/metadata.csv:127]:
```suggestion
stardog.databases.dbname.dictionary.dict.memtable.unpinned.size.bytes,gauge,,byte,,The current size of all unpinned memtables for the dictionary encoding table for the database. Unpinned memtables are memtables which are not currently pinned in memory for readers.,0,stardog,unpinned size bytes,

```

File: stardog/metadata.csv [hunk 6]
```diff
@@ -1,507 +1,512 @@
...
+stardog.databases.dbname.binary.memtable.total.size.bytes,gauge,,byte,,"The current size of all memtables (active, inactive, and immutable), in bytes.",0,stardog,total size bytes,
+stardog.databases.dbname.binary.memtable.unpinned.size.bytes,gauge,,byte,,The current size of all unpinned memtables for the binary count index. Unpinned memtables are memtables which are not currently pinned in memory for readers.,0,stardog,unpinned size bytes,
+stardog.databases.dbname.binary.numFilesCompacting,gauge,,unit,,The current number of files compacting for the binary count index.,0,stardog,dbname binary numFilesCompacting,
+stardog.databases.dbname.binary.numKeys,gauge,,unit,,"The estimated number of keys in the binary count index. For Ternary indices, this is a rough estimate of the number of quads in the database, for the dictionary, it's an estimate of how many statements are in the dictionary. Note that this _not_ a transactional estimate: deleted entries are ignored, so this value will likely overcount in that scenario.",0,stardog,dbname binary numKeys,
+
```
Comment (urseberry) [inline stardog/metadata.csv:48]:
```suggestion
```
Remove extra line, unless it's needed.

**Review by urseberry — APPROVED** [review]
Besides the extra line break, looks good to go!

**Pull request merged**